### PR TITLE
Add total time to test_differ in etc/ci/performance

### PR DESCRIPTION
**Pull request by asajeffrey**
Add the total time to test_differ in etc/ci/performance.

---

- [X] `./mach build -d` does not report any errors
- [X] `./mach test-tidy` does not report any errors
- [X] These changes do not require tests because this is test infrastructure

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/19206)

### CODE REVIEW

File: etc/ci/performance/test_differ.py [hunk 1]
```diff
@@ -49,4 +53,13 @@ def load_data(filename):
         diff = value2 - value1
         change = diff / value1
         color = BLUE if value1 <= value2 else GREEN
+        total1 += value1
+        total2 += value2
         print("{}{:6} {:6} {:+6} {:+8.2%}   {}.{}".format(color, value1, value2, diff, change, key, END))
+
+
+print
```
Comment (edunham) [inline etc/ci/performance/test_differ.py]: Testing locally, this bare print appears to insert a blank line in python2 but it's a no-op in python3. You need `print()` to get the newline.

Comment (asajeffrey) [reply etc/ci/performance/test_differ.py]: (Bashes head on desk). Fixed.

File: etc/ci/performance/test_differ.py [hunk 2]
```diff
@@ -49,4 +53,13 @@ def load_data(filename):
         diff = value2 - value1
         change = diff / value1
         color = BLUE if value1 <= value2 else GREEN
+        total1 += value1
+        total2 += value2
         print("{}{:6} {:6} {:+6} {:+8.2%}   {}.{}".format(color, value1, value2, diff, change, key, END))
+
+
+print()
```
Comment (aneeshusa) [inline etc/ci/performance/test_differ.py]: nit: If the intent is to print a empty line, I would explicitly put an empty string between the parens.

File: etc/ci/performance/test_differ.py [hunk 3]
```diff
@@ -49,4 +53,13 @@ def load_data(filename):
...
+diff = total2 - total1
+change = diff / total1
+color = BLUE if total1 <= total2 else GREEN
+print("{}{:6} {:6} {:+6} {:+8.2%}   {}.{}".format(color, total1, total2, diff, change, "TOTAL", END))
```
Comment (aneeshusa) [inline etc/ci/performance/test_differ.py]: It might be nice to pull these 4 lines into a helper.

**Review by aneeshusa — COMMENTED** [review]
r=me with or without nits

File: etc/ci/performance/test_differ.py (hunk 2 above)
Comment (asajeffrey) [reply etc/ci/performance/test_differ.py]: Fixed.

File: etc/ci/performance/test_differ.py (hunk 3 above)
Comment (asajeffrey) [reply etc/ci/performance/test_differ.py]: Fixed.

**Review by asajeffrey — COMMENTED** [review]
Addressed comments.

**Comment by jdm** [comment]
@bors-servo: retry
* #13480

**Pull request merged**